src/ops/fuelup_self.rs

```diff
@@ -18,6 +22,38 @@ pub fn attempt_install_self(download_cfg: DownloadCfg, dst: &Path) -> Result<()>
     Ok(())
 }
 
+const GOODBYE_HEADER: &str = "Thanks for hacking in Sway!";
+const GOODBYE_INFO: &str = "This will uninstall all Sway toolchains and data, and remove, $HOME/.fuelup/bin from your PATH environment variable.";
```

Comment:
```suggestion
const GOODBYE_INFO: &str = "This will uninstall all Sway toolchains and data, and optionally remove $HOME/.fuelup/bin from your PATH environment variable.";
```